src/main/java/seedu/address/model/person/Fees.java

```diff
@@ -7,18 +7,18 @@
  * Different from payment class as this represents the action of student making the payment
  * Guarantees: immutable;
  */
-public class FeesPaidByStudent {
+public class Fees {
 
-    public static final String MESSAGE_CONSTRAINTS = "Fees paid must be an positive integer amount";
+    public static final String MESSAGE_CONSTRAINTS = "Fees must be an positive integer amount";
     public static final String VALIDATION_REGEX = "^[0-9]\\d*|0$\n";
     public final String value;
 
     /**
-     * Constructs an {@code FeesPaidByStudent}.
+     * Constructs an {@code Fees}.
      *
      * @param fees String for payment status
      */
-    public FeesPaidByStudent(String fees) {
+    public Fees(String fees) {
```

Comment:
Could we use int to store fees instead?

Reply:
I also think maybe it would be more straightforward if fees was stored as an int instead? Since we need to use the value for calculations anyway, it could be easier to directly access the int of converting the string everytime

Reply:
Think I used string cause it takes less steps to transition from user input to fees, and from fees to the GUI
The only part where it have to do arithmetic operations are during calculateFees, which contains the parser to int